opentelemetry/deployment/README.md

```diff
@@ -0,0 +1,25 @@
+# Deployment
+
+Overall, this configuration sets up an OpenTelemetry collector deployed as a Deployment that receives Jaeger, OTLP, OpenCensus and Zipkin traces, adds Kubernetes attributes, and exports the traces to Tempo via OTLP.
+
+## How to run
+1. Deploy the resources from the `otelcol.yaml` manifest file:
+    ```sh
```

Comment:
indentation mistake?

Reply:
Since this was a list, adding this indentation keeps the command under the element of the list.